hip-lang-config.cmake.in

```diff
@@ -65,17 +65,15 @@ find_dependency(amd_comgr)
 
 include( "${CMAKE_CURRENT_LIST_DIR}/hip-lang-targets.cmake" )
 
-# From hip-lang config directory, do three level up
-get_filename_component(_DIR "${CMAKE_CURRENT_LIST_DIR}" REALPATH)
-get_filename_component(_IMPORT_PREFIX "${_DIR}/../../../" REALPATH)
+# Find the hip-lang config file path with symlinks resolved
+# RealPath: /opt/rocm-ver/lib/cmake/hip-lang/hip-lang-config.cmake
+# Go 4 level up to get IMPORT PREFIX
+get_filename_component(_DIR "${CMAKE_CURRENT_LIST_FILE}" REALPATH)
+get_filename_component(_IMPORT_PREFIX "${_DIR}/../../../../" ABSOLUTE)
 
-
-#need _IMPORT_PREFIX to be set #FILE_REORG_BACKWARD_COMPATIBILITY
-file(GLOB HIP_CLANG_INCLUDE_SEARCH_PATHS "${_IMPORT_PREFIX}/../llvm/lib/clang/*/include")
-file(GLOB HIP_CLANG_INCLUDE_SEARCH_PATHS_REORG "${_IMPORT_PREFIX}/llvm/lib/clang/*/include")
+file(GLOB HIP_CLANG_INCLUDE_SEARCH_PATHS "${_IMPORT_PREFIX}/llvm/lib/clang/*/include")
```

Comment:
@gregrodgers This is what I mentioned about hip-lang never done properly. The hard coded `llvm` on the path clearly choked aomp. The correct way should locate the clang header files inlcude path based on the compiler not the full rocm root. This is actually done correctly below using HIP_COMPILER_INSTALL_PATH for CLANG_RT.

I gave the fix as https://github.com/ROCm-Developer-Tools/HIP/pull/2776/files
But the upstream keeps doing wrong and even worse.